### ISSUE TITLE
doesnt work on RTX 2060

### ISSUE THREAD

**Issue by FPcrab** [alievk/avatarify-desktop#29]
E5 2678v3 32GB RAM, RTX 2060
app runs, make calibration, when i click to image below - app crash witn no error.

**Comment by JohanAR** [comment]
If you run the .exe from a terminal (powershell or cmd) does it tell you anything?

Did you add your own photos to the app? If yes, try removing them and only use the default avatars

**Comment by FPcrab** [comment]
no, i did not add custom images. Terminal also shows nothing. AFter execute exe - and after rporgam window apperance - it release console terminal.

**Comment by JohanAR** [comment]
Sorry I don't know much about debugging things on Windows

**Comment by USMC85** [comment]
Anyone ever get this to work with the RTX cards?  Mine does exactly what everyone described here.

**Comment by saiyi123** [comment]
It seems that all RTX graphics cards are not supported. RTX2060 cannot be used, and all RTX30XX graphics cards cannot be used.

**Comment by JohanAR** [comment]
Other people have successfully run it on RTX2060 after following the default installation instructions (i.e. not using any workarounds or patches). But I suppose it's possible that some dependency has changed.
